src/components/HomePage/CreateWorkspaceModal/index.tsx

```diff
@@ -7,6 +7,10 @@ import TextField from '@shared/TextField';
 
 import { ColorVariant } from '@models/styles';
 import { appText } from '@utils/strings';
```

Comment:
Theese two should be above the import useStyles from './styles'. In the same group, at the bottom

import { ColorVariant } from '@models/styles';
import { appText } from '@utils/strings';
 import useStyles from './styles'.   like this